classes/data-stores/ActionScheduler_DBStore.php

```diff
@@ -932,7 +932,7 @@ protected function claim_actions( $claim_id, $limit, \DateTime $before_date = nu
 		 *
 		 * @param string $order_by_sql
 		 */
-		$order    = apply_filters( 'action_scheduler_claim_actions_order_by', 'ORDER BY priority ASC, attempts ASC, scheduled_date_gmt ASC, action_id ASC' );
+		$order    = apply_filters( 'action_scheduler_claim_actions_order_by', 'ORDER BY priority ASC, attempts ASC, scheduled_date_gmt ASC, action_id ASC', $claim_id, $hooks );
```

Comment:
I wonder if it would be cleaner (in terms of further possible future modifications) to pass additional information via an array). For instance:

```php
$properties = array(
	'claim_id' => $claim_id,
	'hooks'    => $hooks,
);

$order = apply_filters( 
	'action_scheduler_claim_actions_order_by', 
	'ORDER BY priority ASC, attempts ASC, scheduled_date_gmt ASC, action_id ASC', 
	$properties
);
```

We should also make sure the docblock reflects the available parameters, whether or not we adopt that suggestion.

Reply:
On reflection, let's stick with your current approach. It's probably worth adding an *additional* [`@since`](https://docs.phpdoc.org/guide/references/phpdoc/tags/since.html) tag, though. Something like:

```php
 * @since 3.8.3 Made $claim_id and @hooks available.
```